src/linagora.esn.unifiedinbox/views/sidebar/email/menu.pug

```diff
@@ -25,7 +25,8 @@
         div(ng-if="displayPersonnalFolders.length > 0")
           hr
           div {{ 'My folders' | translate }}
-          mailbox-display(feature-flag="inbox.experiment.tree-folders", feature-flag-hide, is-folder="true", hide-aside="$hide()", mailbox="mailbox", ng-repeat="mailbox in mailboxes | filter: { role: { value: '!' }, namespace: { type: 'Personal' }, name: ctrl.filter } track by mailbox.id")
+          div(ng-repeat="folder in displayPersonnalFolders track by folder.id")
```

Comment:
Did you test also the tree display and the search feature ? without the filter in th ng-repeat `name: ctrl.filter` I think this will not work

Reply:
the tree display is handled in another component `mailbox-display-tree`.
for the filter it is already applied in the controller so I used the filtered list `displayPersonnalFolders` instead of refiltering `mailboxes` in the template